src/Castle.Windsor.Tests/Diagnostics/DuplicatedDependenciesDiagnosticTestCase.cs

```diff
@@ -31,9 +31,6 @@ public class DuplicatedDependenciesDiagnosticTestCase : AbstractContainerTestCas
 		protected override void AfterContainerCreated()
 		{
 			var host = Kernel.GetSubSystem(SubSystemConstants.DiagnosticsKey) as IDiagnosticsHost;
-#if SILVERLIGHT
-			host.AddDiagnostic<IDuplicatedDependenciesDiagnostic>(new DuplicatedDependenciesDiagnostic(Kernel));
-#endif
```

Comment:
Oops, brain fart. My mind was obviously not reading the colours right after reviewing so many conditionals 😄 , I was thinking that the middle line was left behind. All good 👍 .